Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/bean/ConceptsMergeBean.java

```diff
@@ -0,0 +1,164 @@
+package edu.asu.conceptpower.app.bean;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import edu.asu.conceptpower.app.core.POS;
+import edu.asu.conceptpower.core.ConceptType;
+
+public class ConceptsMergeBean {
```

Comment:
@jdamerow  I created this new class inorder to easily bind the values to the front end. Once user clicks on prepare merge concepts a UI screen appears with the merged concept details. So I thought we could have a new bean that maps to this screen and user can edit the values of the merged concept. Should I rename this new bean class file ?